packages/volto/news/6629.bugfix

```diff
@@ -0,0 +1 @@
+Fixed Relations controlpanel when have broken relations. @giuliaghisini
```

Comment:
I grabbed your comment, and amended the log to make it clear how someone can identify the problem.

```suggestion
Correctly handle when one of the items is `null` in the Relations control panel. @giuliaghisini
```